src/api/app/views/webui/request/beta_show_tabs/_conversation.html.haml

```diff
@@ -33,7 +33,38 @@
   .col-md-8.order-md-1.order-sm-2
     %h4.list-group.mb-4 Comments & Request History
     .timeline{ data: { comment_counter: local_assigns[:comment_counter_id] } }
+      .timeline-item
+        .d-inline-flex
+          - creator = User.find_by_login(bs_request.creator) || User.nobody
+          = image_tag_for(creator, size: 35, custom_class: 'rounded-circle bg-light border border-gray-400 avatars-counter')
+          %p
+            %i.fas.fa-code-commit.text-dark
+            = link_to(realname_with_login(creator), user_path(creator))
+            created this request
+            = link_to('#request-creation', title: l(bs_request.created_at.utc), name: 'request-creation') do
+              #{time_ago_in_words(bs_request.created_at)} ago
+            - if bs_request.superseding.any?
+              superseding
+              - bs_request.superseding.each do |superseded_request|
+                = link_to("request ##{superseded_request.number}", request_show_path(superseded_request.number, anchor: 'overview'))
+
+      - bs_request.superseding.each do |superseding_request|
+        .timeline-item
+          = link_to('#collapse-superseding',
+              data: { toggle: 'collapse' },
+              aria: { expanded: false, controls: 'collapse-superseding' }) do
+            %i.timeline-break.expander>
+            %i.timeline-offset.collapser>
+
+            %i.fas.fa-fw.fa-chevron-right.expander{ title: 'Show collapse' }>
```

Comment:
The title is not really needed here since we have the text right next to this icon. Same for the icon on the next line. If you prefer to keep the titles, I suggest changing _collapse_ to _history_:

```suggestion
            %i.fas.fa-fw.fa-chevron-right.expander{ title: 'Show history' }>
```